sdk/cosmosdb/cosmos/src/client/Diagnostics/Diagnostic.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { cosmosDiagnosticsLogger, setCosmosDiagnosticsLogLevel } from "../../utils/logger";
```

Comment:
We are creating a client logger for business customers.  You can see the log levels don't match.
/**

The log levels supported by the Cosmos Diagnostics.
The log levels are:
verbose
DataPlaneRequests - Not implemented
QueryRuntimeStatistics - Not implemented
PartitionKeyStatistics - Not implemented
ControlPlaneRequests - Not implemented
*/

Example 1 - basic usage
const { EventHubClient } = require('@azure/event-hubs');

const logger = require('@azure/logger');
logger.setLogLevel('info');

// operations will now emit info, warning, and error logs
const client = new EventHubClient(/* params */);
client.getPartitionIds()
  .then(ids => { /* do work */ })
  .catch(e => { /* do work */ });
});

it is tested here 
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/logger/test/logger.spec.ts

describe("ClientLoggers", () => {
  it("logs to parent loggers", () => {
    Logger.setLogLevel("verbose");

    const oldLog = Logger.AzureLogger.log.bind(Logger.AzureLogger);
    let called = false;

    Logger.AzureLogger.log = () => {
      called = true;
    };

    testLogger.info("hello");
    assert.ok(called);

    Logger.AzureLogger.log = oldLog;
  });

Reply:
Okay, but what about this scenario can't be supported by the existing logger? Why do we need custom functions that set the verbose logging level and log to stdout instead of stderr?

Can the customer not simply use the `AZURE_LOG_LEVEL` environment variable, or the `DEBUG` environment variable to selectively enable and disable the client logger's namespace?

I don't see how the test code relates. It is just testing that logger extension works as expected. It's not a model for client or customer code.